package.json

```diff
@@ -80,12 +82,14 @@
     "eslint-plugin-prettier": "^5.1.3",
     "fetch-intercept": "^2.4.0",
     "husky": "^9.0.11",
+    "i": "^0.3.7",
```

Comment:
The `i` and `npm` installs look like they might have been the result of a CLI typo.